poetry.lock

```diff
@@ -475,15 +471,14 @@ files = [
 name = "importlib-metadata"
 version = "6.7.0"
 description = "Read metadata from Python packages"
-optional = false
+optional = true
```

Comment:
Do you know why this changed? (I just ran `poetry lock --no-update` to satisfy the pre-commit check.)

Reply:
i always just run `poetry lock` why do need `--no-update` flag?

Reply:
To keep the PR focussed on dropping Python 3.7 and (I hope) not change any other behavior.

I used `--no-update` for its documented behavior:

> Do not update locked versions, only refresh lock file.

A dependency update may cause unintended changes in Sceptre.

We can use a dedicated PR to update the dependencies.